### ISSUE TITLE
Remove sys.maxint in twisted/spread

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8444]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8444|
|Type|enhancement|
|Created|2016-06-10 00:15:48Z|
|Branch|https://github.com/twisted/twisted/tree/8444-rodrigc-maxint|

In Python 3, sys.maxint is gone:
https://docs.python.org/3/whatsnew/3.0.html#integers

and will give an attribute error.

sys.maxsize can be used on Python 2 and 3 instead.

<details><summary>Searchable metadata</summary>

```
trac-id__8444 8444
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8444_rodrigc_maxint 8444-rodrigc-maxint
branch_author__ 
status__closed closed
resolution__fixed fixed
component__pb pb
keywords__None None
time__1465517748883564 1465517748883564
changetime__1466479046362804 1466479046362804
version__None None
owner__rodrigc rodrigc
cc__warner
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/165

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

Sent to buildbot

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_5"></a>|@wsanchez set owner to @rodrigc|
|-|-|

Reviewed, looks good.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_6"></a>|@rodrigc set status to `closed`|
|-|-|

In changeset 07d6fc8d1febeff7fc3dfb7eddbd51554f47cad8
```
#!CommitTicketReference repository="" revision="07d6fc8d1febeff7fc3dfb7eddbd51554f47cad8"
Merge '8444-rodrigc-maxint': Replace sys.maxint with sys.maxsize

Author: rodrigc
Reviewer: wsanchez
Fixes: #8444
```
